waiter/test/waiter/core_test.clj

```diff
@@ -434,7 +434,9 @@
         (let [request {:request-method :delete, :uri (str "/apps/" service-id), :authorization/user user}
               {:keys [body headers status]} (async/<!! ((ring-handler-factory waiter-request?-fn handlers) request))]
           (is (= 200 status))
-          (is (= {"content-type" "application/json"} headers))
+          (is (= {"content-type" "application/json"
```

Comment:
These look like standard headers, repeated about 40 times.

Why not: 

(def expected-error-response-headers {....})
Then reference that dictionary throughout?

Reply:
Introduced a new variable to hold the expected headers and left the assertions in the tests.